EIPS/eip-7702.md

```diff
@@ -27,11 +27,12 @@ There is a lot of interest in adding short-term functionality improvements to EO
 
 ### Parameters
 
-|     Parameter        | Value  |
-| -------------------- | ------ |
-| `SET_CODE_TX_TYPE`   | `0x04` |
-| `MAGIC`              | `0x05` |
-| `PER_AUTH_BASE_COST` | `2500` |
+|     Parameter             | Value  |
+| ------------------------- | ------ |
+| `SET_CODE_TX_TYPE`        | `0x04` |
+| `MAGIC`                   | `0x05` |
+| `PER_AUTH_BASE_COST`      | `4600` |
```

Comment:
Note the updated cost per auth tuple, 2500 → 4600, due to adding one `COLD_SLOAD_COST`.